### ISSUE TITLE
Could not reproduce results on ModelNet40

### ISSUE THREAD

**Issue by charlesCXK** [lulutang0608/Point-BERT#37]
Dear author,
Thanks for your sharing! I am trying to reproduce the results with the given checkpoint ([Tsinghua Cloud](https://cloud.tsinghua.edu.cn/f/202b29805eea45d7be92/?dl=1)). Here are my results:

|Dataset|Best Acc|Best Acc (vote)|
|--|--|--|
|ModelNet40 (1k points)| 92.38/92.26 | 92.71/92.66 |
|ModelNet40 (4k points)| 92.99 | 93.11 |
|ModelNet40 (8k points)| 93.31 | 93.59 |

For ModelNet40 (1k points), I have run it twice and both the two results are lower than the number you report (Acc=92.67, Vote Acc=93.24). Here are my questions:
1. I am using PyTorch 1.7 with CUDA 11.0 on a single A100 GPU. For other packages, I follow your instructions and install them accordingly. **I could obtain similar or slightly higher performance on ScanObjectNN**. Could you successfully reproduce the performance of ModelNet40 (1K) and ModelNet40 (4K)?
2. I find that, a checkpoint with the best accuracy doesn't mean it has the best vote accuracy. Does these two metrics (Acc & Vote Acc) are selected from different checkpoints?

Thanks and hoping for your reply!

**Comment by yuxumin** [comment]
Hi, thanks for your interest in our work.
For these two questions:
- Yes, i can reproduce the performance on ModelNet40 ignoring the small deviation. I trained the models on three different type of GPUs, i.e. Nvidia 2080 Ti, Nvidia 3090 and Tesla A100. We find that the performance on Nvidia 3090 is the best (And our performance reported in the paper is produced with Nvidia 3090). We recommend you to reproduce the results with Nvidia 3090.
- We only select one checkpoint to test the metric. It is unpractice to test acc_vote for many models in experiments.

Hi, thanks for your interest in our work.
For these two questions:
- Yes, I can reproduce performance on ModelNet40 ignoring small deviations. I trained the model on three different types of GPUs, i.e. Nvidia 2080 Ti, Nvidia 3090 and Tesla A100. We found the performance to be the best on the Nvidia 3090 (and the performance we report in the paper was produced using the Nvidia 3090). We recommend that you use Nvidia 3090 to reproduce the results.
- We choose only one checkpoint to test. It is impractical to test `acc_vote` for many models.
